src/user-guide/suite-storage-etc.rst

```diff
@@ -3,14 +3,14 @@
 Suite Storage, Discovery, Revision Control, and Deployment
 ==========================================================
 
-Small groups of cylc users can of course share suites by manual copying,
-and generic revision control tools can be used on cylc suites as for any
-collection of files. Beyond this cylc does not have a built-in solution
-for suite storage and discovery, revision control, and deployment, on a
-network. That is not cylc's core purpose, and large sites may have
-preferred revision control systems and suite meta-data requirements that
-are difficult to anticipate. We can, however, recommend the use of
-*Rose* to do all of this very easily and elegantly with cylc suites.
+Cylc suites should be portable and may be version controlled, however, beyond
+this Cylc does not have a built-in solution for suite storage and discovery,
```

Comment:
```suggestion
Cylc does not have a built-in solution for suite storage and discovery,
``` 
("should be portable"?)

Reply:
I didn't feel that this was adding anything but can add it back.